React.podspec

```diff
@@ -114,6 +114,12 @@ Pod::Spec.new do |s|
     ss.preserve_paths   = "Libraries/LinkingIOS/*.js"
   end
 
+  s.subspec 'RCTLinking' do |ss|
+    ss.dependency         'React/Core'
+    ss.source_files     = "Libraries/Linking/*.{h,m}"
```

Comment:
There is no native code living in Linking folder. Everything is in `LinkingIOS`.